docs/index.md

```diff
@@ -14,15 +14,17 @@ Nerve is a tool for building LLM agents using a simple YAML-based syntax. Agents
 
 ### Installing with Cargo
 
-The easiest and recommended way to install Nerve is via [Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html):
+The easiest and recommended way to install Nerve is with [Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html):
 
 ```sh
 cargo install nerve-ai
 ```
 
 ### Installing from DockerHub
 
-Alternatively a Docker image is available on [Docker Hub](https://hub.docker.com/r/dreadnode/nerve). In order to run it, keep in mind that you'll probably want the same network as the host in order to reach the OLLAMA server, and remember to share in a volume the tasklet files:
+Alternatively, a Docker image is available on [Docker Hub](https://hub.docker.com/r/dreadnode/nerve). In order to run it, you'll probably want the same network as the host in order to reach the OLLAMA server, and remember to share in a volume the tasklet files:
+
+<!-- This snippet needs a bit more clarity. Perhaps splitting the sentences would help? The piece about "remember to share in a volume the tasklet files" is a little confusing. Does the user need to share the network? I think I need some help understanding this piece! -->
```

Comment:
When you run a docker container, it is isolated in its own network by default - this prevents exposing anything on the host (your computer) network to it. 

However, for some use cases (using local inference servers like ollama for instance) Nerve _needs_ to connect to those network endpoints, therefore the --network=host part below means that the container will run without network isolation and will "see" the same things your computer "sees" on the network.

The user doesn't need to do anything more than using this command line. I hope it makes more sense now!